docs/_guide/properties.md

```diff
@@ -387,68 +387,46 @@ Generated accessors automatically call `requestUpdate`, initiating an update if
 
 ### Create custom property accessors {#accessors-custom}
 
-To specify how getting and setting works for a property, create custom accessors:
+To specify how getting and setting works for a property, create custom accessors instead of using automatically generated accessors:
 
 ```js
 // Declare a property
 static get properties() { return { myProp: { type: String } }; }
 
 // Custom accessors
-set myProp(value) { ... /* Custom setter */ } 
+set myProp(value) {
+  // Capture old value
+  const oldValue = this.myProp;
+  // Implement custom setter logic
+  ...
+  // Pass the old value to request an update
+  this.requestUpdate('myProp', oldValue);
+} 
 get myProp() { ... /* Custom getter */ }
 
 ...
 
 // Later, set the property
-this.myProp = 'hi'; // Invokes generated accessor, which calls custom accessor
+this.myProp = 'hi'; // Invokes custom accessor
 ```
 
-When you create custom property accessors for a property, LitElement still generates its own accessors unless you specify otherwise ([see below](#accessors-noaccessor)). The generated setter:
-
-* Saves the previous property value.
-* Calls your custom setter.
-* Requests an update, supplying the property name and its old value to the update lifecycle.
+If your class defines custom property accessors for a declared property directly on its prototype (i.e., not inherited from a superclass), LitElement will not overwrite them with generated accessors.  Such custom setters must manually request an update, supplying the property name and its old value to the update lifecycle by calling `requestUpdate`.
 
 ### Prevent LitElement from generating a property accessor {#accessors-noaccessor}
 
-To prevent LitElement from generating property accessors, set `noAccessors` to `true` in the property declaration:
+To prevent LitElement from generating property accessors, set `noAccessor` to `true` in the property declaration:
```

Comment:
Should we clarify that `noAccessor` is not needed if you're written your own accessors?